controller.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'view'
+require_relative 'flashcard'
+require_relative 'deck'
+require_relative 'card_parser'
+
+require 'pry'
+
+class Controller
+  include Parsable
+  attr_reader :new_deck
+
+  def initialize
+    @new_deck = Deck.new(parse_from_txt('source_cards.txt'))
+    @view = View.new
+    @view.introduction
+    @view.user_interface
+    run_interface
+  end
+
+  def run_interface
+    random_number = rand(0..@new_deck.flashcards.length)
```

Comment:
This should not be the concern of the controller. One hint / indicator / smell that that's the case is that fact that you are reaching through the deck, into the flashcards and asking the length (breaking the Law of Demeter).

I would expect this work to be done on the Deck itself. I want to be able to send a message to the deck like: `deck.random_card`